src/App.js

```diff
@@ -1,15 +1,76 @@
 import React from 'react';
 import { Switch, BrowserRouter as Router, Route } from 'react-router-dom';
+import { useAuth0 } from '@auth0/auth0-react';
 import LandingPage from './pages/LandingPage';
+import LoggedInConsumer from './pages/LoggedInConsumer';
+import LoggedInContractor from './pages/LoggedInContractor';
+import ForConsumers from './pages/ForConsumers';
+import ForContractors from './pages/ForContractors';
 
 const App = () => {
-    return (
-        <Router>
-            <Switch>
-                <Route exact path="/" component={LandingPage}></Route>
-            </Switch>
-        </Router>
-    );
+    const { isAuthenticated, isLoading, user } = useAuth0();
+
+    //NOTE: REFACTOR THESE IF STATEMENTS
+    if (isLoading) {
+        return <div>loading 123...</div>;
+    }
+    if (!isAuthenticated) {
+        return (
+            <Router>
+                <Switch>
+                    <Route exact path="/" component={LandingPage}></Route>
+                    <Route
+                        exact
+                        path="/ForConsumers"
+                        component={ForConsumers}
+                    ></Route>
+                    <Route
+                        exact
+                        path="/ForContractors"
+                        component={ForContractors}
+                    ></Route>
+                </Switch>
+            </Router>
+        );
+    }
+    if (
```

Comment:
all this logic should happen in a anoother compoent...ill show you how to do this dont worry about it for now